tkg/clusterclient/credentials.go

```diff
@@ -443,6 +443,37 @@ func (c *client) GetAzureCredentialsFromSecret() (azureclient.Credentials, error
 	return res, nil
 }
 
+func (c *client) GetAzureCredentialsFromIdentity(identityName string, identityNamespace string) (azureclient.Credentials, error) {
+	var (
```

Comment:
Declare these two variables directly when we use them?

Reply:
Done!